<launch>
  <arg name="output_log" default="false" />
  <arg name="leaf_size" default="0.5" />
  <arg name="measurement_range" default="100" />
  <arg name="input_topic_name" default="points_raw" />
  <arg name="output_topic_name" default="filtered_points"/>
  
  <node pkg="points_downsampler" name="voxel_grid_filter" type="voxel_grid_filter">
    <param name="input_topic_name" value="points_raw" />
    <param name="output_topic_name" value="filtered_points" />
    <param name="leaf_size" value="$(arg leaf_size)" />
    <param name="measurement_range" value="$(arg measurement_range)" />
    <param name="output_log" value="$(arg output_log)" />
  </node>
</launch>
